index.html

```diff
@@ -87,163 +85,165 @@ <h3 class="work-subtitle">Multi-Post Stories</h3>
+          <ul class="lang-list2">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <div class="prj-link butt1"><a class="prj-link-a" href="#">See project</a></div>
+        </div>
+        <div class="other-works">
+          <h3 class="card-subtitle2">Profesional Art Printing Data</h3>
+          <p class="card-text2">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+            been the industry's standard</p>
+          <ul class="lang-list2">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <div class="prj-link"><a class="prj-link-a" href="#">See project</a></div>
+        </div>
+        <div class="other-works">
+          <h3 class="card-subtitle2">Profesional Art Printing Data</h3>
+          <p class="card-text2">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+            been the industry's standard</p>
+          <ul class="lang-list2">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <div class="prj-link"><a class="prj-link-a" href="#">See project</a></div>
+        </div>
+        <div class="other-works">
+          <h3 class="card-subtitle2">Profesional Art Printing Data</h3>
+          <p class="card-text2">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+            been the industry's standard</p>
+          <ul class="lang-list2">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <div class="prj-link"><a class="prj-link-a" href="#">See project</a></div>
+        </div>
+        <div class="other-works">
+          <h3 class="card-subtitle2">Profesional Art Printing Data</h3>
+          <p class="card-text2">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+            been the industry's standard</p>
+          <ul class="lang-list2">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <div class="prj-link"><a class="prj-link-a" href="#">See project</a></div>
         </div>
       </div>
-    </section>
-    <section class="about-me">
+    </div>
+  </section>
+  <section class="about-me">
+    <div class="abt-text-container">
       <h2 class="abt-title2">About me</h2>
       <p class="abt-text2">Hello I’m a software developer! I can help you build a product , feature or website Look
-        through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate
+        through some of my work and experience! If you like what you see and have a project you need coded, don’t
+        hestiate
         to contact me.</p>
       <div class="resume"><a class="resume-a" href="#">Get My Resume</a></div>
```

Comment:
- [ ] Kindly add button interaction styles to this button like you did wit the others 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/213731030-f8964bcb-b0b2-425a-9ca5-501c86876f35.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/213731088-7a299209-a011-4a0f-8628-52cf70e679fe.png)